guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>After training <s>in case of</s> the <code>FullyConnectedNetwork</code> , model the train vs valid losses can be plotted using <code>fcn.plot_losses()</code> to check if the model is overfitting. For example, the plot below shows that the model has <u>been </u>trained well. <s>and though</s> While the losses are still gradually decreasing, <s>but</s> they are not decreasing at a significant level <s>not significantly</s>, <s>hence</s> and the model is <u>therefore </u>ready and can be saved for inferencing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>